### DISCUSSION TITLE
伟大的花火大人很伤心！

### DISCUSSION THREAD

**Discussion by ybbgh** [share121/inter-knot#3362] [其他]
伟大的花火大人昨日号召人们按下烟花按钮，可过去了一天，人数没有达到。
伟大的花火大人期待已久的烟花没有绽放，她非常伤心！<img src="https://bbs-static.miyoushe.com/static/2024/01/31/42b5ef8bab5a58fbdd74f17c21c12420_2665774360840406961.png" alt="花火 略略略" width="75" >
因此，花火大人在六分街放置了10个炸弹！如果今天找不出来的话，整个六分街都会被炸上天！
对了，伟大的花火大人还有一个温馨提示：有999个炸弹，里面只有十个是真的哦！
![无标题](https://github.com/user-attachments/assets/542e5db7-ce0b-4ef8-ba57-be6289b25a20)
这是炸弹，努力寻找吧<img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" >

**Comment by tayiie** [comment]
## 千万别按，我昨天按完就被大屁股捉去治安署了，
`虽说里面的汉堡酥酥麻麻的味道不错`，
可隔壁的熊希人还是太恐怖了，那个关在一起小邦布的转头都被他抢过来掏耳朵了，小邦布都哭了
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/651fba00ba55004182fffbb5aa553c53_6086077287436441482.png" alt="米仔_膜拜" width="75" >

**Reply by ybbgh** [reply]
这可是炸弹哦！花火大人给了你们一次机会，好好珍惜吧！去努力寻找炸弹吧<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火 礼物" width="75" >

**Comment by wyqsagiri** [comment]
在一个星空里的列车中看到了好多个，都按下了

**Reply by ybbgh** [reply]
按钮已经没用了！花火大人非常伤心，但给了你们一次机会！只要找到10个炸弹并解除，新艾利都就不会被炸上天哦！快去吧<img src="https://bbs-static.miyoushe.com/static/2024/01/31/cf527198a404a4db566dc4a9d68befc3_2947652818567379105.png" alt="花火 嘿嘿" width="75" >

**Comment by Chilin987** [comment]
偷偷溜出来打游戏，在电玩店找到了10个 这是怎么绕过店主藏进去的啊？？？
有一个居然是明目张胆摆在游戏机顶上的，这都没有被发现嘛<img src="https://img-static.mihoyo.com/communityweb/upload/12a80a8f5da3949b086c497b8cc4b5d3.png" alt="安比_头痛" width="75" >
 ~~~ 会不会找完发现是1009个啊（ ~~~

**Reply by ybbgh** [reply]
嘿嘿，这是秘密哦
